pyproject.toml

```
@@ -190,6 +190,7 @@ extend-exclude = [
   "util/vendor.py",
   "util/verible-format.py",
   "util/version_file.py",
+  "**/pwrmgr/util/*.py",
```

```suggestion
  "hw/ip_templates/pwrmgr/util/*.py",
```
To avoid dealing with replicates.

---

It is important to exclude the replicates because they are copies and therefore will fail the linting as well.